CMakeLists.txt

```diff
@@ -557,8 +552,10 @@ unset(Z3_SECOND_PACKAGE_INCLUDE_DIR)
 unset(Z3_CXX_PACKAGE_INCLUDE_DIR)
 unset(AUTO_GEN_MSG)
 unset(CONFIG_FILE_TYPE)
-# TODO: Provide a `Z3Version.cmake` file so that clients can specify the version
-# of Z3 they want.
+
+write_basic_package_version_file("${PROJECT_BINARY_DIR}/Z3ConfigVersion.cmake"
+  COMPATIBILITY SameMajorVersion
```

Comment:
Can be changed to ``AnyNewerVersion``, if backwards compatibility across major releases is planned.

Reply:
noted.